### PR TITLE
[PROPIM-596] New FlexibilityIcon

### PR DESCRIPTION
**Pull request by Tuxosaurus**
Add "flexibilityIcon".

Screen Shot 2020-12-01 at 14 03 39

Tested locally in Firefox MacOs.

### CODE REVIEW

**Comment by wakooka** [comment]
Can you point me to the icon in the Figma? Can't find it

**Comment by Tuxosaurus** [comment]
> Can you point me to the icon in the Figma? Can't find it

Here, all the other icons are already available: https://www.figma.com/file/wz8HAFFvfLckH1w9j6f4YL/USP-Bus-LP-icons?node-id=0%3A1

**Review by wakooka — APPROVED** [review]

**Comment by wakooka** [comment]
> https://www.figma.com/file/wz8HAFFvfLckH1w9j6f4YL/USP-Bus-LP-icons?node-id=0%3A1

Thanks, it should be added to the Pixar icons, do you know if it's planned?

**Comment by Tuxosaurus** [comment]
> > Here, all the other icons are already available: https://www.figma.com/file/wz8HAFFvfLckH1w9j6f4YL/USP-Bus-LP-icons?node-id=0%3A1
> 
> Thanks, it should be added to the Pixar icons, do you know if it's planned?

I'll ask

**Review by gabrielagabriel — CHANGES_REQUESTED** [review]
Hello @Tuxosaurus following your conversation with @wakooka and new guidelines on adding  new components to Pixar, we must validate the presence of this icon on the library before implementing here. 
Thank you in advance.

File: src/icon/flexibilityIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export const FlexibilityIcon = (props: Icon) => (
+  <BaseIcon {...props} viewBox="0 0 24 24">
+    <g fill={props.iconColor} fillRule="evenodd">
```
Comment (gabrielagabriel) [inline src/icon/flexibilityIcon.tsx:7]: Do we really need the `g `since we have only one `path`?

**Comment by Tuxosaurus** [comment]
> Hello @Tuxosaurus following your conversation with @wakooka and new guidelines on adding new components to Pixar, we must validate the presence of this icon on the library before implementing here.
> Thank you in advance.

It has been added to Pixar

File: src/icon/flexibilityIcon.tsx (hunk 1 above)
Comment (Tuxosaurus) [reply src/icon/flexibilityIcon.tsx]: Thanks, indeed, I modified the SVG I received by removing a useless path but didn't see the now lonely <g>

**Review by gabrielagabriel — APPROVED** [review]

**Pull request merged**